### PR TITLE
[DOCS-7077] Update and reorg main SDS doc

### PR DESCRIPTION
**Pull request by maycmlee**
### What does this PR do? What is the motivation?

- Removes the old SDS doc
- Add new SDS folder and new doc (as `_index`)
- Adds updated images

[DOCS-7077](https://datadoghq.atlassian.net/browse/DOCS-7077)

### Merge instructions

- [ ] Please merge after reviewing

### Additional notes

[DOCS-7077]: https://datadoghq.atlassian.net/browse/DOCS-7077?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/sensitive_data_scanner/_index.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+{{% /tab %}}
+{{< /tabs >}}
+
+**Notes**:
+
+- Any rules that you add or update only affect data coming into Datadog after the rule was defined.
+- Sensitive Data Scanner does not affect any rules you define on the Datadog Agent directly.
+- To turn off Sensitive Data Scanner entirely, set the toggle to **off** for each Scanning Group and Scanning Rule so that they are disabled.
+
+See [Investigate Sensitive Data Issues][7] for details on how to use the [Summary][8] page to triage your sensitive data issues.
```
Comment (maycmlee) [inline content/en/sensitive_data_scanner/_index.md:124]: This link will work after [this PR](https://github.com/DataDog/documentation/pull/21567) is merged into the `may/main-sds-refresh` branch.

File: content/en/sensitive_data_scanner/_index.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+- [Adding scanning rules](#add-scanning-rules)
+- [Controlling access to events with sensitive data](#control-access-to-events-with-sensitive-data)
+
+You can also [redact sensitive data in tags](#redact-sensitive-data-in-tags).
+
+### Add a scanning group
+
+A scanning group determines what data to scan. It consists of a query filter and a set of toggles to enable scanning for Logs, APM, RUM, and Events. See the [Log Search Syntax][3] documentation to learn more about query filters.
+
+For Terraform, see the [Datadog sensitive data scanner group][4] resource.
```
Comment (cswatt) [inline content/en/sensitive_data_scanner/_index.md:59]:
```suggestion
For Terraform, see the [Datadog Sensitive Data Scanner group][4] resource.
```

File: content/en/sensitive_data_scanner/_index.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+
+To set up a scanning group:
+
+1. Navigate to [Sensitive Data Scanner][5]. Alternatively, go to Organization Settings > Sensitive Data Scanner.
```
Comment (cswatt) [inline content/en/sensitive_data_scanner/_index.md:63]:
```suggestion
1. Navigate to [Sensitive Data Scanner][5]. Alternatively, go to **Organization Settings** > **Sensitive Data Scanner**.
```

File: content/en/sensitive_data_scanner/_index.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+
+- [Adding a scanning group](#add-a-scanning-group)
+- [Adding scanning rules](#add-scanning-rules)
+- [Controlling access to events with sensitive data](#control-access-to-events-with-sensitive-data)
+
+You can also [redact sensitive data in tags](#redact-sensitive-data-in-tags).
+
+### Add a scanning group
+
+A scanning group determines what data to scan. It consists of a query filter and a set of toggles to enable scanning for Logs, APM, RUM, and Events. See the [Log Search Syntax][3] documentation to learn more about query filters.
```
Comment (cswatt) [inline content/en/sensitive_data_scanner/_index.md:57]:
```suggestion
A scanning group determines what data to scan. It consists of a query filter and a set of toggles to enable scanning for logs, APM, RUM, and events. See the [Log Search Syntax][3] documentation to learn more about query filters.
```

File: content/en/sensitive_data_scanner/_index.md [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+
+For Terraform, see the [Datadog sensitive data scanner group][4] resource.
+
+To set up a scanning group:
+
+1. Navigate to [Sensitive Data Scanner][5]. Alternatively, go to Organization Settings > Sensitive Data Scanner.
+1. Click **Add scanning group**. Alternatively, click the **Add** dropdown menu on the top right corner of the page and select **Add Scanning Group**.
+1. Enter a query filter for the data you want to scan. At the top, click **APM Spans** to preview the filtered spans. Click **Logs** to see the filtered logs.
+1. Enter a name and description for the group.
+1. Click the toggle buttons to enable Sensitive Data Scanner for the products you want (for example, Logs, APM Spans, RUM Events, and Events).
```
Comment (cswatt) [inline content/en/sensitive_data_scanner/_index.md:67]:
```suggestion
1. Click the toggle buttons to enable Sensitive Data Scanner for the products you want (for example, logs, APM spans, RUM events, and general events).
```

There may be a better way to say "general events", but it just stuck out to me to say "RUM events and events"

File: content/en/sensitive_data_scanner/_index.md [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+{{% sds-scanning-rule %}}
+1. Click **Add Rules**.
+
+{{% /tab %}}
+{{% tab "Add a custom scanning rule" %}}
+
+You can create custom scanning rules using regex patterns to scan for sensitive data.
+
+1. In the **Define match conditions** section, specify the regex pattern to use for matching against events in the **Define regex** field. Enter sample data in the **Regex tester** field to verify that your regex pattern is valid.   
+    Sensitive Data Scanner supports Perl Compatible RegEx (PCRE), but the following patterns are not supported:
```
Comment (cswatt) [inline content/en/sensitive_data_scanner/_index.md:101]:
```suggestion
    Sensitive Data Scanner supports Perl Compatible Regular Expressions (PCRE), but the following patterns are not supported:
```

File: content/en/sensitive_data_scanner/_index.md [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+    - Backreferences and capturing sub-expressions (lookarounds)
+    - Arbitrary zero-width assertions
+    - Subroutine references and recursive patterns
+    - Conditional patterns
+    - Backtracking control verbs
+    - The \C "single-byte" directive (which breaks UTF-8 sequences)
```
Comment (cswatt) [inline content/en/sensitive_data_scanner/_index.md:107]:
```suggestion
    - The `\C` "single-byte" directive (which breaks UTF-8 sequences)
```

File: content/en/sensitive_data_scanner/_index.md [hunk 8]
```diff
@@ -0,0 +1,178 @@
...
+    - The \R newline match
```
Comment (cswatt) [inline content/en/sensitive_data_scanner/_index.md:108]:
```suggestion
    - The `\R` newline match
```

File: content/en/sensitive_data_scanner/_index.md [hunk 9]
```diff
@@ -0,0 +1,178 @@
...
+    - The \K start of match reset directive
```
Comment (cswatt) [inline content/en/sensitive_data_scanner/_index.md:109]:
```suggestion
    - The `\K` start of match reset directive
```

File: layouts/shortcodes/sds-scanning-rule.md [hunk 10]
```diff
@@ -0,0 +1,8 @@
+1. For **Create keyword dictionary**, add keywords to refine detection accuracy when matching regex conditions. For example, if you are scanning for a sixteen-digit Visa credit card number, you can add keywords like `visa`, `credit`, and `card`. You can also require that these keywords must be within a specified number of characters of a match. By default, keywords must be within 30 characters before a matched value.
+1. For **Define actions on match**, select the action you want to take for the matched information. **Note**: Redaction, partial redaction, and hashing are all irreversible actions.
+    - **Redact**: Replaces all matching values with the text you specify in the **Replacement text** field.
+    - **Partially Redact**: Replaces a specified portion of all matched data. In the **Redact** section, specify the number of characters you want to redact and which part of the matched data to redact.
+    - **Hash**: Replaces all matched data with a unique identifier. The UTF-8 bytes of the match is hashed with the 64-bit fingerprint of farmhash.
```
Comment (cswatt) [inline layouts/shortcodes/sds-scanning-rule.md:5]:
```suggestion
    - **Hash**: Replaces all matched data with a unique identifier. The UTF-8 bytes of the match is hashed with the 64-bit fingerprint of FarmHash.
```

**Review by cswatt — CHANGES_REQUESTED** [review]

File: content/en/sensitive_data_scanner/_index.md (hunk 5 above)
Comment (maycmlee) [reply content/en/sensitive_data_scanner/_index.md:67]: Thanks, updated to `Datadog events`

**Review by cswatt — APPROVED** [review]

**Pull request merged**